src/homework-04/indexes_discriminated.ts

```diff
@@ -0,0 +1,64 @@
+// 1.
+// Визначте інтерфейс, який використовує сигнатуру індексу з типами об'єднання.
+// Наприклад, тип значення для кожного ключа може бути число | рядок.
+
+interface ITask1 {
+  [key: number]: string;
+  [key: string]: string;
```

Comment:
The union type should be used for the value, not for the key. I couldn't find any union type, though. Please be cautious :-)

Reply:
Fixed  task1